src/less/filter-button/base/filter-button.less

```diff
@@ -0,0 +1,67 @@
+button.filter-button,
+a.filter-link {
+    .filter-button-base;
+
+    background-color: @filter-button-background-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-hover-background-color;
+    }
+}
+
+.filter-button__cell,
```

Comment:
Should we prefix this with button and a tags? Or can this be added to non buttons/anchors
Like
```
button.filter-button__cell,
a.filter-button__cell
```
Only for consistency since I see it on all the others.